### PR TITLE
Minor adjustments to make it easier for the reader.

### PR DESCRIPTION
**Pull request by ridalalven**
Hi Manuel, I just fixed some parts in the "Get started" part that I found outdated. I am not to good at git; therefore, I will send some more adjustments in another commit.

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -114,11 +109,11 @@ Don't change the default file name, which should be *hello_world.json*.
 Just select a folder from your system (it can be the same where you saved the main program file)
 and click **Save**.
 
-Now you should see the new tool specification in the *Project* widget, *Tool specifications* list.
+Now you should see the new tool specification in the *Project* widget, *Tools* list.
```
Comment (manuelma) [inline docs/source/getting_started.rst:112]: I think this should be: 

    Now you should see the new tool in the *Project* widget, *Tools* list.

Can  you make that change as part of this PR @ridalalven ? After that I think we are ready to merge. Thanks very much for grooming this!

**Comment by PekkaSavolainen** [comment]
These files are now available again:

/spinetoolbox/ui/resources/file-link.svg
/spinetoolbox/ui/resources/project_item_icons/hammer.svg
/spinetoolbox/ui/resources/wrench.svg

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -175,13 +172,11 @@ the *Process Log* or in the *Python Console* depending on your settings (See :re
...
+|
 
-.. note::
-   If you encounter the following message in Event Log when trying to execute a Python Tool.
-
-   **Couldn't determine Python version. Please check the Python interpreter option in Settings.**
-
-   Please see :ref:`Setting Up External Tools` for help.
+.. note:: If you encounter the message **Couldn't determine Python version. Please check the Python interpreter option in Settings.**
+   in the *Event Log* when trying to execute a Python Tool, check :ref:`Setting Up External Tools` for help.
```
Comment (PekkaSavolainen) [inline docs/source/getting_started.rst:179]: This ref should now be

:ref:`Setting up External Tools`

Comment (PekkaSavolainen) [reply docs/source/getting_started.rst:179]: Nevermind. The link works anyway, Sorry for the noise.

**Comment by manuelma** [comment]
Thanks for reviewing @PekkaSavolainen. Is this ok with you? I think we should merge.

**Comment by PekkaSavolainen** [comment]
Sure, go for it.

**Pull request merged**